### PR TITLE
Rework sidebar 237

### PR DESCRIPTION
**Pull request by tylerpar99**
This PR fixes issue #237 

## Summary:
The sidebar structure had become outdated due to new changes to development. Now the sidebar will have an admin drop-down that will include Activity Logs (prev. Admin Logs), Course Proposals (prev. Course Proposal Management), and Settings (prev. Admin Management). This drop-down is not accessible as a Student Admin.  

## To Test:
Go to the CELTS web app as an admin and you will see the admin dropdown and the options held under it. Pay close attention to what happens after you click as the tab will remain actively highlighted but the drop-down will close and hide the options this can easily be reopened but I am not sure your feelings regarding this. Also, the clickable area is noticeably smaller than the regular nav links. Not sure if these differences are okay or if you would like to approach this differently.

## Previous UI:
image

### CODE REVIEW

File: app/logic/events.py [hunk 1]
```diff
@@ -119,11 +112,19 @@ def getStudentLedProgram(term):
...
+                                        .order_by(fn.COUNT(1).desc()))
+                                        .where(Event.term == term).get())
+    trainingEvents = ((Event.select(Event)
+			               .order_by((Event.id == allTrainingsEvent.event.id).desc(), Event.startDate.desc())
                            .where(Event.isTraining,
-                                  Event.term == term))
+                                  Event.term == term)))
+
+    for t in trainingEvents:
+        print(t)
```
Comment (BrianRamsay) [inline app/logic/events.py]: debug output

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
Let's add a few nice things. 
1. the Admin menu should load in an expanded position if we are on one of the pages in the submenu. 
2. an open/closed indicator would be nice (or at least a closed one) so it is obvious that it is expandable.
3. there's a bit of a gap between the items in the submenu and the Admin header

**Review by BrianRamsay — APPROVED** [review]
Haha, wow, you changed those things a lot faster than I thought you would!

File: app/logic/events.py (hunk 1 above)
Comment (tylerpar99) [reply app/logic/events.py]: I think this disappeared when I merged development in

**Pull request merged**